### PR TITLE
Fix types in subscribeAllToAll roster GraphQL API

### PR DESCRIPTION
**Pull request by jacekwegr**
This PR fixes types in subscribeAllToAll roster GraphQL API and adds tests for them

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/136075) / small_tests / 996efa53e85f0bd970690d385805e31fc1dac58a
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3756/136075/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3756/136075/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-09-09_14.59.15/lib.mongooseim.logs/run.2022-09-09_14.59.15/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/136135) / small_tests / 0da283d67197257c76b0e2ed256a460aaf3746c4
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3756/136135/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3756/136135/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-09-12_11.58.18/lib.mongooseim.logs/run.2022-09-12_11.58.19/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/136201) / small_tests / 28161cce07b54a03e0ab5b433a0b1bd0379f017e
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3756/136201/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3756/136201/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-09-12_13.30.16/lib.mongooseim.logs/run.2022-09-12_13.30.17/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/136251) / small_tests / 2e58f7cdaade8806afe34f8260000bddfda8ef6d
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3756/136251/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3756/136251/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-09-12_14.46.22/lib.mongooseim.logs/run.2022-09-12_14.46.22/suite.log.html)

File: big_tests/tests/graphql_roster_SUITE.erl [hunk 1]
```diff
@@ -314,6 +323,24 @@ admin_subscribe_to_all_with_wrong_user_story(Config, Alice, Bob) ->
     check_contacts([Bob], Alice),
     check_contacts([Alice], Bob).
 
+
```
Comment (chrzaszcz) [inline big_tests/tests/graphql_roster_SUITE.erl:326]: This empty line is unnecessary.

**Review by chrzaszcz — APPROVED** [review]
Looks good 👍

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/136634) / small_tests / a08a82e5896813f036b4b3da3290b0e651bca694
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3756/136634/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3756/136634/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-09-14_09.43.36/lib.mongooseim.logs/run.2022-09-14_09.43.36/suite.log.html)

**Pull request merged**